python/example/single_cell_detailed.py

```diff
@@ -21,15 +21,11 @@
 
 # (2) Create and populate the label dictionary.
 
-labels = arbor.label_dict()
+# Pre-defined SWC regions
+labels = arbor.label_dict(swc=True)
```

Comment:
Let's make it a bit clearer this is setting defaults:
```suggestion
labels = arbor.label_dict(default="swc")
```
or 

```suggestion
labels = arbor.label_dict(defaults_swc=True)
```
(there may be other defaults?)

Reply:
No, these are the only ones we have. Other _loaders_ return a structure containing labels and morphology, but for SWC it's essentially always the same. You're correct in that passing a boolean is not really the high point of design here. At the very least it should be kwargs-only, but I think it would be nice to have something to add SWC labels post-hoc, st
```py
ld = A.label_dict({....}).add_swc_labels()
```
will give you the labels you explicitly passed _plus_ our SWC defaults.

Reply:
Done.